tests/consensus_tests/test_io_uring_eintr.py

```
@@ -0,0 +1,201 @@
+                    "top": 10,
+                    "with_vector": False,
+                },
+                timeout=5,
+            )
+            results.append(("ok", r.status_code))
+        except requests.exceptions.ConnectionError:
+            results.append(("connection_error", None))
+            break
+        except requests.exceptions.ReadTimeout:
+            results.append(("timeout", None))
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)